applications/serial_lte_modem/doc/FOTA_AT_commands.rst

```diff
@@ -47,10 +49,17 @@ Syntax
 
 .. note::
 
-   When doing modem FOTA, erasing the modem DFU area is optional since the update process will automatically erase the area if needed.
+   When doing modem delta FOTA, erasing the modem DFU area is optional since the update process will automatically erase the area if needed.
```

Comment:
```suggestion
   During a delta update to the modem over FOTA, the modem DFU area is automatically erased as needed as part of the update process. Erasing this area manually is optional.
```

Trying to make the structure a little clearer. I hope I understood this correctly.

Reply:
Fixed